Java/libraries/recognizers-text/src/main/java/com/microsoft/recognizers/text/ResolutionKey.java

```diff
@@ -9,4 +11,9 @@
     public static final String IsoCurrency = "isoCurrency";
     public static final String Offset = "offset";
     public static final String RelativeTo = "relativeTo";
+    public static final String Timex = "timex";
+    public static final String Start = "start";
+    public static final String End = "end";
+    public static final String Mod = "mod";
+    public static final String SourceEntity = "sourceEntity";
```

Comment:
These values should be removed, these are not related to the `ResolutionKey`.

These are related to the DateTimeResolutionKey and they are already added in [DateTimeResolutionKey](https://github.com/microsoft/Recognizers-Text/blob/master/Java/libraries/recognizers-text-date-time/src/main/java/com/microsoft/recognizers/text/datetime/DateTimeResolutionKey.java) following [C#](https://github.com/microsoft/Recognizers-Text/blob/08b488ae1c83127eb93b26d381b8d668b548a620/.NET/Microsoft.Recognizers.Text.DateTime/DateTimeResolutionKey.cs)